cobaya/cosmo_input/input_database.py

```diff
@@ -479,10 +479,16 @@
 
 like_bao = {none: {},
             'BAO_planck_2018': {
```

Comment:
I think that for this we should create a new BAO set set to the latest comprehensive data combination available (`BAO_last`, or sth like that?) and preserve the Planck 2018 BAO set for archival purposes. What do you both think?

Reply:
Yes that sounds like a good idea